src/helperFunctions/beautifyTaskStatus.ts

```diff
@@ -0,0 +1,19 @@
+import task from '../interfaces/task.type';
+
+const beautifyTaskStatus = (tasks: Array<task>) => {
+  const taskList: Array<task> = [];
+  tasks.forEach((taskItem: task, taskIndex) => {
+    taskList.push(taskItem);
+    const splitedStringArray = taskItem.status.split('_');
+    splitedStringArray.forEach((subString, index) => {
+      splitedStringArray[index] = subString.toLowerCase();
+    });
+    splitedStringArray.forEach((string, index) => {
+      splitedStringArray[index] = string.charAt(0).toUpperCase() + string.slice(1);
+    });
+    taskList[taskIndex].status = splitedStringArray.join(' ');
```

Comment:
I feel like it can be handled in a single line provided the condition is how it is defined

```
const string = "aaaaa_masasdaas"

const convertData = (input) => input.toLowerCase().split('_').map(item => item.charAt(0).toUpperCase() + item.slice(1)).join(' ');

convertData(string)
console.log(convertData(string))
```

I was just trying it out , let me know your view on how readable or difficulty faced in reading.
But something to look into to make it more concise

Reply:
I feel like it can be handled in a single line provided the condition is how it is defined

```
const string = "pawan_is_a_meMBER_oF_RDS"

const convertData = (input) => input.toLowerCase().split('_').map(item => item.charAt(0).toUpperCase() + item.slice(1)).join(' ');

convertData(string)
console.log(convertData(string))
```

I was just trying it out , let me know your view on how readable or difficulty faced in reading.
But something to look into to make it more concise